bookshelf/api_v3/cloud_instance.py

```diff
@@ -7,10 +7,12 @@ class Distribution(Enum):
     Enumeration of distributions supported by the bookshelf v3 api.
 
     :ivar CENTOS7: constant for CentOS 7.
-    :ivar UBUNTU1404: constante for Ubuntu LTS 14.04.
+    :ivar UBUNTU1404: constant for Ubuntu LTS 14.04.
+    :ivar UBUNTU1604: constant for Ubuntu LTS 16.04.
```

Comment:
It's really weird that we adopted this scheme where every variable has to be documented in two places, in the value thereof and in here to explain the semantics -- which is partially what led to the misspelling of the word "constant" in one of them.

Is the only requirements for the constants to be distinct or do they have to equal something else?
If distinct, let's put all the information in the token and make it more readable:

    UBUNTU1404 = u"Ubuntu LTS 14.04"

If you need it to be a valid symbol, just substitute with _ like

    UBUNTU1404 = u"Ubuntu_LTS_14_04"

(which should be good enough to avoid needing the explanation above).

Then the explanation in the doc string can explain the scheme: "Each constant is a short name for the distribution. The value describes the distribution in more detail for human beings (so that the value will get printed out in log and error messages, for example)."

This is not a deal breaker -- either ignore this, fix it or create a ticket :)